counter-strike2config.json

```diff
@@ -0,0 +1,212 @@
+[
+    {
+        "DisplayName":"Server Name",
+        "Category":"CS2 Server Settings",
+        "Description":"Sets the name of the server that appears in the server list",
+        "Keywords":"server,name,hostname",
+        "FieldName":"hostname",
+        "InputType":"text",
+        "ParamFieldName":"hostname",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"AMP Powered Counter-Strike 2 Server",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Server Tags",
+        "Category":"CS2 Server Settings",
+        "Description":"Comma-separated list of server tags that give additional information about the server in the server browser",
+        "Keywords":"tags,sv_tags",
+        "FieldName":"sv_tags",
+        "InputType":"text",
+        "ParamFieldName":"sv_tags",
+        "IncludeInCommandLine":true,
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"LAN Server",
+        "Category":"CS2 Server Settings",
+        "Description":"If enabled, the server will operate as a LAN server: no heartbeat, no authentication, no non-class C addresses, 9999.0 rate, etc",
```

Comment:
It's unclear what 9999.0 rate means here. Is that the network speed?

Reply:
I will remove that reference. It is a hangover from the srcds description of the sv_lan setting and is not referenced in the help text for the CS2 cvar, so can be removed.